assets/app/view/game/game_info.rb

```diff
@@ -149,16 +149,21 @@ def upcoming_trains
             end
           end
 
-          h(:tr, [
-            h(:td, names_to_prices.keys.join(', ')),
-            h('td.right', names_to_prices.values.map { |p| @game.format_currency(p) }.join(', ')),
-            h(:td, trains.size),
-            h(:td, obsolete_schedule[name]&.join(', ') || 'None'),
-            h(:td, rust_schedule[name]&.join(', ') || 'None'),
-            h(:td, discounts&.join(' ')),
-            h(:td, train.available_on),
-            h(:td, event_text.join(', ')),
-          ])
+          upcoming_train_content = []
```

Comment:
change it to 

upcoming_trains_content = [
a, 
b, 
c,
]

upcoming_train << x if y

ucoming_train.concat([
d,
e,
f]

Reply:
can you make it 

upcoming_train_content = [
...
]